gempython/tests/optohybrid_info_uhal.py

```diff
@@ -45,105 +45,51 @@
-            readRegister(ohboard,"GEM_AMC.OH.OH%d.CONTROL.HDMI_OUTPUT.SBITS"%(options.gtx)),
-            clocking["refclock"])
-
-    print "Lock status:  %8s  %7s  %4s  %3s"%("FPGA PLL","EXT PLL","CDCE","GTX")
-    print "                   0x%x      0x%x   0x%x  0x%x"%(
-            clocking["fpgaplllock"],
-            clocking["extplllock"],
-            clocking["cdcelock"],
-            clocking["gtxreclock"])
-    pass
-else:
-    print "Sources:  %6s  %7s  %9s"%("TrgSrc","SBitSrc","RefCLKSrc")
-    print "             0x%x      0x%x        0x%x"%(
-            readRegister(ohboard,"GEM_AMC.OH.OH%d.CONTROL.TRIGGER.SOURCE"%(options.gtx)),
-            readRegister(ohboard,"GEM_AMC.OH.OH%d.CONTROL.HDMI_OUTPUT.SBITS"%(options.gtx)),
-            clocking["refclock"])
-
-    print "Lock status:  %10s  %13s"%("QPLL","QPLL FPGA PLL")
-    print "                     0x%x            0x%x"%(
-            clocking["qplllock"],
-            clocking["qpllfpgaplllock"])
-    errorCounts = nesteddict()
-    errorCounts["QPLL"] = []
-    errorCounts["FPGA"] = []
-    for trial in range(options.errorRate):
-        errorCounts["QPLL"].append(calculateLockErrors(ohboard,options.gtx,"QPLL",SAMPLE_TIME))
-        errorCounts["FPGA"].append(calculateLockErrors(ohboard,options.gtx,"QPLL_FPGA_PLL",SAMPLE_TIME))
-        pass
-    qrates = getErrorRate(errorCounts["QPLL"],SAMPLE_TIME)
-    frates = getErrorRate(errorCounts["FPGA"],SAMPLE_TIME)
-    print "Unlock count: 0x%08x     0x%08x"%(qrates[0],frates[0])
-    print "Unlock rate:%10sHz   %10sHz"%("%2.2f%s"%(qrates[1],qrates[2]),"%2.2f%s"%(frates[1],frates[2]))
+print "Lock status:  %10s  %13s"%("GBT MMCM","LOGIC MMCM")
+print "                     0x%x            0x%x"%(
+    clocking["gbtmmcmlock"],
+    clocking["logicmmcmlock"])
+errorCounts = nesteddict()
+print "Unlock count: 0x%08x     0x%08x"%(
+    clocking["gbtmmcmlockcnt"],
+    clocking["logicmmcmlockcnt"])
+
 print
-#print "-> OH Clocking (src, bkp):     VFAT         CDCE"
-#print "-> %22s%d       (0x%x  0x%x)   (0x%x  0x%x)"%("link",links[link],
-#						     clocking["vfatsrc"],clocking["vfatbkp"],
-#						     clocking["cdcesrc"],clocking["cdcebkp"])
 #exit(1)
 
-if (options.resetCounters):
-    optohybridCounters(ohboard,options.gtx,True)
-    pass
-
 print "-> Counters    %8s     %8s     %8s     %8s"%("L1A","Cal","Resync","BC0")
-counters = optohybridCounters(ohboard,options.gtx)
-for key in counters["T1"]:
-    print "   %8s  0x%08x   0x%08x   0x%08x   0x%08x"%(key,
-                                                       counters["T1"][key]["L1A"],
-                                                       counters["T1"][key]["CalPulse"],
-                                                       counters["T1"][key]["Resync"],
-                                                       counters["T1"][key]["BC0"])
-    pass
+# counters = optohybridCounters(ohboard,options.gtx)
```

Comment:
It didn't work, and 90% of the content was removed for V3 and I didn't bother to print that info.
It can be added in if requested.
Caveat @bdorney, if you want more to be printed by these info scripts, let me know.
My goal was to get the scripts not crashing, and showing some amount of useful info for V3

Reply:
if we don't add them, let's just remove this part altogether...

Reply:
I was not using them and not sure how useful the remaining counters are.  Since this is legacy we can probably drop this entirely.

Reply:
Then I'd delete them in legacy branch (e.g. here)